apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/EnkelPersonTrygdetid.tsx

```diff
@@ -78,11 +78,12 @@ export const EnkelPersonTrygdetid = (props: Props) => {
     return (
       <VStack gap="12" maxWidth="69rem">
         <TrygdetidManueltOverstyrt
-          behandlingId={behandling.id}
+          behandling={behandling}
```

Comment:
Lurer på om det er bedre å ikke sende inn denne, og heller bruke `useBehandling` i komponenten

Reply:
For å unngå unødvendige params, eller?